Add Docker + GitHub recipe (#121)
PrefectHQ/prefect-recipes: README.md

- [Deploy Flows to AWS](https://discourse.prefect.io/t/how-to-deploy-prefect-2-0-flows-to-aws/1252)
- [Deploy Flows to GCP](https://discourse.prefect.io/t/how-to-deploy-prefect-2-0-flows-to-gcp/1251)
- [Deploy Flows to Azure](https://discourse.prefect.io/t/how-to-deploy-prefect-2-0-flows-to-azure/1312)

> - [Use Docker Container as an Infrastructure and GitHub as a Storage](https://towardsdatascience.com/create-robust-data-pipelines-with-prefect-docker-and-github-12b231ca6ed2)

Reviewer: ```suggestion
- [Store Flows in GitHub and Execute in a Docker Container](https://towardsdatascience.com/create-robust-data-pipelines-with-prefect-docker-and-github-12b231ca6ed2)
```

Reviewer: I think something like the suggestion I made above would be more in line with the existing naming conventions for the recipes so far :)
